### ISSUE TITLE
WCAG violation: Special character in title

### ISSUE THREAD

**Issue by ZuperNova** [Altinn/app-frontend-react#1395]
### Description of the bug

According to WCAG 2.1 SC 2.4.2:
The page's `<title>` element should not contain special characters, such as `~` or `^`.
 
![image-2023-06-15-15-27-43-970](https://github.com/Altinn/app-frontend-react/assets/5908907/89e19bd0-76ec-45be-b7dc-8eb906c7e27b)
![image-2023-06-15-15-28-17-382](https://github.com/Altinn/app-frontend-react/assets/5908907/8edabfd2-c7f4-4a5a-ad93-a02ef5b8ff71)

### Steps To Reproduce

No need for steps, frontend adds the special character.
![image](https://github.com/Altinn/app-frontend-react/assets/5908907/470b2682-2500-4caf-8e0c-26eb949f41b7)

**Comment by Febakke** [comment]
Yes, i suggest we swap it with a "-" as in the third example from [technique G88](https://www.w3.org/WAI/WCAG21/Techniques/general/G88)

As a sidenote, i cant seem to find any information that this is a part of 2.4.2 Page title. Do you know where it comes from?

**Comment by ZuperNova** [comment]
@Febakke We had a "UU"-tester run through our application and used the software called [ARC ToolKit](https://chrome.google.com/webstore/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce). It was this software that discovered the violation.
